.dprint.json

```diff
@@ -1,30 +0,0 @@
-{
-  "typescript": {
-    "deno": true
-  },
-  "markdown": {
-    "deno": true
-  },
-  "json": {
-    "deno": true
-  },
-  "exec": {
-    "cwd": "${configDir}",
-    "commands": [{
-      "command": "rustfmt",
-      "exts": ["rs"]
-    }]
-  },
```

Comment:
This was ensuring that Rust code was formatted too and now it's lost. Can you add the check to the CI script?